tests/unit/ReportActionsUtilsTest.js

```diff
@@ -74,7 +74,31 @@ describe('ReportActionsUtils', () => {
                     message: [{html: 'Hello world'}],
                 },
                 {
-                    actionName: CONST.REPORT.ACTIONS.TYPE.CLOSED,
+                    actionName: CONST.REPORT.ACTIONS.TYPE.CREATED,
+                    message: [{html: 'Hello world'}],
+                },
+                {
+                    actionName: CONST.REPORT.ACTIONS.TYPE.IOU,
+                    message: [{html: 'Hello world'}],
+                },
+                {
+                    actionName: CONST.REPORT.ACTIONS.TYPE.RENAMED,
+                    message: [{html: 'Hello world'}],
+                },
+                {
+                    actionName: 'REIMBURSED',
+                    message: [{html: 'Hello world'}],
+                },
+            ];
+            const result = ReportActionsUtils.filterReportActionsForDisplay(input);
+            input.pop();
+            expect(result).toStrictEqual(input);
+        });
+
+        it('should filter out closed actions', () => {
```

Comment:
Added a new unit test to check if a closed action is removed and updated the existing one that had a closed action.